### PR TITLE
MSUIF Accordion component

### PR DESCRIPTION
**Pull request by damianmuti**
This is a CSS-only accordion component that uses CSS :target event and HTML anchors to show or hide content.

### CODE REVIEW

**Comment by lcalvino** [comment]
:+1:

File: assets/styles/modules/_accordions.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+$accordion-item-padding:         10px;
+
+.c-accordion {
+  @at-root {
+    &__item {
+      max-height: $accordion-header-height;
+      overflow: hidden;
+      border-bottom: 1px solid tint($accordion-header-background, 20%);
+      background-color: $accordion-item-background;
+      transition: max-height 0.3s ease 0s;
```
Comment (nazarenooviedo) [inline assets/styles/modules/_accordions.scss]: A better approach would be use a variable to control transition time in all framework, with this implementation, all transitions would be consistent with each other.

Comment (damianmuti) [inline assets/styles/modules/_accordions.scss]: I thought about that while coding both components. That's a **great** idea. Will do the changes right now. :+1:

Comment (nazarenooviedo) [inline assets/styles/modules/_accordions.scss]: :+1:  :)

**Pull request merged**